test/CalculateWinner.test.ts

```diff
@@ -0,0 +1,132 @@
+import { test } from "good-vibes";
```

Comment:
None of these tests are actually running, because it has not been imported into `all.test.ts` file. Can you please import it and add that file to this merge request also?

Reply:
![Screenshot 2022-10-15 at 12 52 52 PM](https://user-images.githubusercontent.com/4240966/195974618-3a43a47a-b741-48b0-ad0d-6e15bc9f1d70.png)